Week3/homework/step2-3.js

```diff
@@ -31,8 +31,9 @@ function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  do {
```

Comment:
What happens if `num <=0`? The `do-while` loop executes at least once because the loop condition is only tested at the end of the loop. This makes the `do-while` loop rather useless: it can always be rewritten with a regular (safe) `while` loop.

But since we asked you here to use a `do-while` loop, what code can you add to take care of `num <= 0`?

(My previous comment about the complex loop condition applies here too.)